src/client/search/SearchTermsReducer.spec.js

```diff
@@ -0,0 +1,63 @@
+import { should } from 'chai';
+import deepFreeze from 'deep-freeze';
+import SearchTermsReducer from './SearchTermsReducer';
+
+describe(SearchTermsReducer, () => {
+  it('should add a hashtag search term', () => {
+    const stateBefore = [];
+    const action = {
+      type: 'ADD_SEARCH_TERM',
+      id: 0,
+      query: 'Football',
+      paramTypes: ['hashtag'],
+      source: 'twitter',
+    };
+
+    const stateAfter = [{
+      id: 0,
+      query: 'Football',
+      paramTypes: ['hashtag'],
+      source: 'twitter',
+    },];
+
+    deepFreeze(stateBefore);
```

Comment:
yes, since it's key when using Redux that the state nor action is mutated, again common practise... if it gets mutated will cause problems in the background